src/pages/Stream/stream.jsx

```diff
@@ -0,0 +1,75 @@
+import { useState } from "react";
+import ReactPlayer from "react-player";
+import { Navbar } from "../../video-components/Navbar/navbar";
+import { Sidebar } from "../../video-components/Sidebar/sidebar";
+import { useParams } from "react-router-dom";
+import "../../root.css"
+import "../Homepage/homepage.css";
+import "../Stream/stream.css";
+import { useVideo } from "../../context/videoContext";
+import { useAuth } from "../../context/authContext";
+
+const Stream = () => {
+const [sidebar, setSideBar] = useState(true);
+const [modal, setModal] = useState(false);
+const {videoState, addItemToLikedVideos, removeItemFromLikedVideos, addItemToWatchLaterVideos, removeItemFromWatchLaterVideos} = useVideo();
+const { videos } = videoState;
+const { videoID} = useParams();
+const { token } =useAuth();
+
+const likeHandler = (token, video) => {
+    videoState.likedVideos.some((item) => item._id === video._id) ?
+    removeItemFromLikedVideos(video._id, token) :
+    addItemToLikedVideos(token,video)
+    }
+
+const watchLaterHandler = ( token, video) => {
+    videoState.watchLater.some((item) => item._id === video._id) ?
+    removeItemFromWatchLaterVideos(video._id, token) : addItemToWatchLaterVideos(token,video)
+    }
+
+const currVideo = videos.find((video) => video._id === videoID)
+console.log("current video",currVideo);
+console.log(videoID);
```

Comment:
You can remove console logs after they're no longer required.

Reply:
Okay